apps/angular/projection/src/app/ui/card/card.component.html

```diff
@@ -1,27 +1,13 @@
-<div
-  class="border-2 border-black rounded-md p-4 w-fit flex flex-col gap-3"
-  [class]="customClass">
-  <img
-    *ngIf="type === CardType.TEACHER"
-    src="assets/img/teacher.png"
-    width="200px" />
-  <img
-    *ngIf="type === CardType.STUDENT"
-    src="assets/img/student.webp"
-    width="200px" />
+<ng-container *ngTemplateOutlet="cardImage.templateRef"></ng-container>
```

Comment:
When there is no logic to give back to the parent or the template you need to project, use ng-content. This is way easier